packages/api/src/routes/settings.ts

```diff
@@ -241,6 +242,31 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /settings/webhook/sample-payload
+ *
+ * Sends a sample payload to the configured webhook URL using the configured webhook key.
+ *
+ * @param payload - The payload to send.
+ * @return {200} indicating sample payload being processed.
+ */
+router.post(
+  "/webhook/sample-payload",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const id = getCxIdOrFail(req);
+    const settings = await getSettingsOrFail({ id });
+    const { webhookUrl, webhookKey } = settings;
```

Comment:
nit, destructure in one line

const { webhookUrl, webhookKey } = await getSettingsOrFail({ id });